### ISSUE TITLE
Installation Issue: petsc

### ISSUE THREAD

**Issue by citibeth** [spack/spack#14052]
### Steps to reproduce the issue

```
==> Staging archive: /gpfsm/dnb53/rpfische/spack7/var/spack/stage/petsc-3.5.3-7kzk4sgwsarxafksa3q3fgrs7hadbcnq/petsc-3.5.3.tar.gz
==> Created stage in /gpfsm/dnb53/rpfische/spack7/var/spack/stage/petsc-3.5.3-7kzk4sgwsarxafksa3q3fgrs7hadbcnq
==> No patches needed for petsc
==> Building petsc [Package]
==> Executing phase: 'install'
==> Error: ProcessError: Command exited with status 1:
    '/usr/bin/python2' 'configure' '--prefix=/gpfsm/dnb53/rpfische/spack7/opt/spack/linux-suse_linux11-x86_64/intel-18.0.4/petsc-3.5.3-7kzk4sgwsarxafksa3q3fgrs7hadbcnq' '--with-ssl=0' '--download-c2html=0' '--download-sowing=0' '--download-hwloc=0' 'CFLAGS=-gcc-name=/usr/local/other/SLES11.3/gcc/5.3.0/bin/gcc' 'FFLAGS=' 'CXXFLAGS=-gxx-name=/usr/local/other/SLES11.3/gcc/5.3.0/bin/g++' '--with-cc=/usr/local/intel/2018/compilers_and_libraries_2018.5.274/linux/mpi/intel64/bin/mpiicc' '--with-cxx=/usr/local/intel/2018/compilers_and_libraries_2018.5.274/linux/mpi/intel64/bin/mpiicpc' '--with-fc=/usr/local/intel/2018/compilers_and_libraries_2018.5.274/linux/mpi/intel64/bin/mpiifort' '--FC_LINKER_FLAGS=-lintlc' '--with-precision=double' '--with-scalar-type=real' '--with-shared-libraries=1' '--with-debugging=0' '--with-64-bit-indices=0' 'COPTFLAGS=' 'FOPTFLAGS=' 'CXXOPTFLAGS=' '--with-blas-lapack-lib=/gpfsm/dnb53/rpfische/spack7/opt/spack/linux-suse_linux11-x86_64/intel-18.0.4/openblas-0.3.5-rkwl3giot3lyziqsjgmjjr7zrnoeibxt/lib/libopenblas.so' '--with-x=0' '--with-clanguage=C' '--with-scalapack-dir=/gpfsm/dnb53/rpfische/spack7/opt/spack/linux-suse_linux11-x86_64/intel-18.0.4/netlib-scalapack-2.0.2-lhdc4cgnnzjy7tb2yw43eklwsggpvnzq' '--with-scalapack=1' '--with-metis=1' '--with-metis-dir=/gpfsm/dnb53/rpfische/spack7/opt/spack/linux-suse_linux11-x86_64/intel-18.0.4/metis-5.1.0-sufypnf66kystdbr4hejp4whexwgoviu' '--with-hdf5=1' '--with-hdf5-dir=/gpfsm/dnb53/rpfische/spack7/opt/spack/linux-suse_linux11-x86_64/intel-18.0.4/hdf5-1.8.18-epkwhxyfopc4w2njmbezd3ve2ynv5eil' '--with-hypre=1' '--with-hypre-dir=/gpfsm/dnb53/rpfische/spack7/opt/spack/linux-suse_linux11-x86_64/intel-18.0.4/hypre-2.13.0-umn5cuuiea7f6g5itxbilfdq5judnray' '--with-parmetis=1' '--with-parmetis-dir=/gpfsm/dnb53/rpfische/spack7/opt/spack/linux-suse_linux11-x86_64/intel-18.0.4/parmetis-4.0.3-vhbzilkc3ovor6yelgr2ttxuss26nxzs' '--with-mumps=1' '--with-mumps-dir=/gpfsm/dnb53/rpfische/spack7/opt/spack/linux-suse_linux11-x86_64/intel-18.0.4/mumps-5.1.1-af54sqrsm2zqfnhmrtzpkvpoaqjp5bzy' '--with-trilinos=0' '--with-superlu_dist=0' '--with-suitesparse=0' '--with-zlib-include=/gpfsm/dnb53/rpfische/spack7/opt/spack/linux-suse_linux11-x86_64/intel-18.0.4/zlib-1.2.11-czw6xbime22i7ergylzvbzwrqov7plxa/include' '--with-zlib-lib=-L/gpfsm/dnb53/rpfische/spack7/opt/spack/linux-suse_linux11-x86_64/intel-18.0.4/zlib-1.2.11-czw6xbime22i7ergylzvbzwrqov7plxa/lib -lz' '--with-zlib=1'

5 warnings found in build log:
     22    TESTING: restoreReconfigure from PETSc.utilities.installDir(/gpfsm/dnb53/rpfische/tmp/spack-stage/spack-stage-1XqlwJ/petsc-3.5.3/config/PETSc/utilities/installDir.py:66)
     23    TESTING: setExternalPackagesDir from PETSc.utilities.externalpackagesdir(/gpfsm/dnb53/rpfische/tmp/spack-stage/spack-stage-1XqlwJ/petsc-3.5.3/config/PETSc/utilities/externalpackagesdir.py:15)
     24    TESTING: cleanExternalpackagesDir from PETSc.utilities.externalpackagesdir(/gpfsm/dnb53/rpfische/tmp/spack-stage/spack-stage-1XqlwJ/petsc-3.5.3/config/PETSc/utilities/externalpackagesdir.py:22)
     25    TESTING: printEnvVariables from config.setCompilers(/gpfsm/dnb53/rpfische/tmp/spack-stage/spack-stage-1XqlwJ/petsc-3.5.3/config/BuildSystem/config/setCompilers.py:1527)
     26    TESTING: resetEnvCompilers from config.setCompilers(/gpfsm/dnb53/rpfische/tmp/spack-stage/spack-stage-1XqlwJ/petsc-3.5.3/config/BuildSystem/config/setCompilers.py:1534)
     27                ===============================================================================
  >> 28    ***** WARNING: CC (set to /gpfsm/dnb53/rpfische/spack7/lib/spack/env/intel/icc) found in environment variables - ignoring
     29                     use ./configure CC=$CC if you really want to use that value ******
     30                ===============================================================================
     31    
     32                ===============================================================================
  >> 33                    ***** WARNING: CXX (set to /gpfsm/dnb53/rpfische/spack7/lib/spack/env/intel/icpc) found in environment variables - ignoring
     34                     use ./configure CXX=$CXX if you really want to use that value ******
     35                ===============================================================================
     36    
     37                ===============================================================================
  >> 38                    ***** WARNING: FC (set to /gpfsm/dnb53/rpfische/spack7/lib/spack/env/intel/ifort) found in environment variables - ignoring
     39                     use ./configure FC=$FC if you really want to use that value ******
     40                ===============================================================================
     41    
     42                ===============================================================================
  >> 43                    ***** WARNING: F77 (set to /gpfsm/dnb53/rpfische/spack7/lib/spack/env/intel/ifort) found in environment variables - ignoring
     44                     use ./configure F77=$F77 if you really want to use that value ******
     45                ===============================================================================
     46    
     47                ===============================================================================
  >> 48                    ***** WARNING: F90 (set to /usr/local/other/SLES11.3/gcc/5.3.0/bin/gfortran) found in environment variables - ignoring
     49                     use ./configure F90=$F90 if you really want to use that value ******
     50                ===============================================================================
     51    
     52    TESTING: checkMPICompilerOverride from config.setCompilers(/gpfsm/dnb53/rpfische/tmp/spack-stage/spack-stage-1XqlwJ/petsc-3.5.3/config/BuildSystem/config/setCompilers.py:1499)
     53    TESTING: requireMpiLdPath from config.setCompilers(/gpfsm/dnb53/rpfische/tmp/spack-stage/spack-stage-1XqlwJ/petsc-3.5.3/config/BuildSystem/config/setCompilers.py:1518)
     54    TESTING: checkVendor from config.setCompilers(/gpfsm/dnb53/rpfische/tmp/spack-stage/spack-stage-1XqlwJ/petsc-3.5.3/config/BuildSystem/config/setCompilers.py:398)

```

**Comment by adamjstewart** [comment]
@citibeth those look like warnings, not something that would cause the build to fail. Can you upload the `spack-build-out.txt`?

**Comment by adamjstewart** [comment]
@balay @BarrySmith @jedbrown

**Comment by balay** [comment]
Did the build fail - or is the issue the warnings printed by PETSc configure?

**Comment by citibeth** [comment]
I’m working on getting this log up, should be able to do so on Wednesday

**Comment by jedbrown** [comment]
Also, is there a reason to be using a 5-year old version?

**Comment by citibeth** [comment]
>
>
> Also, is there a reason to be using a 5-year old version?
>

Welcome to life with coupled models.  First I coupled with PISM; and by the
time I was done, the version I used was a couple of years old.  Then I had
to do some other things on the GCM side. When I got back to PISM, the
version I was using was now ancient.  I wanted to get things back working
with the version of PISM I had previously used, before updating to the
latest PISM; which will invariably involve its own issues.  We have plans
to update "shortly."

>
